sjb/config/test_cases/azure_build_node_image_rhel_310.yml

```diff
@@ -31,6 +34,13 @@ actions:
   - type: "host_script"
     title: "move secrets to remote host"
     script: |-
+      set +x
```

Comment:
Move to file-less way to avoid race conditions:
 ssh -F ./.config/origin-ci-tool/inventory/.ssh_config openshiftdevel "echo 'location_url=${location_url:-}' >> /etc/environment"
or 
>/data/src/github.com/openshift/release/cluster/test-deploy/azure/registry